keyboards/h87a/readme.md

```diff
@@ -0,0 +1,42 @@
+h87a
+===
+
+A QMK-powered replacement PCB for custom TKL keyboards. This PCB has the same size of the b.87 PCB, while using the USB port placement and spacebar stabilizer orientation as the a.87 PCB.
+
+Keyboard Maintainer: hineybush (Josh Hinnebusch)
```

Comment:
Make sure that each of these lines has two spaces on them.

Reply:
For the keyboard maintainer and the next two lines, make sure that each of these has 2 spaces at the end of the line.  Otherwise, it's not formatted correctly.

Reply:
done.